tetris.js

```diff
@@ -241,37 +264,46 @@ module.exports = {
 					let obj = {}
 					if(math[i][j]!= false){
 						if(math[i][j] == "X"){
-							console.log(math[i][j])
 							obj.x = elem_X
 							obj.y = elem_Y
 							quantityElem++;
 							
-							arrayOfCoordinates.push(obj)
+							arrayOfCoordinates.push(obj);
 							
 						}
-						elem_X++
+						elem_X++;
 					}
 					
 				}
-				elem_Y++
+				elem_Y++;
 			}
 			
 		}
+		return arrayOfCoordinates;
+	},
+	
+	buildTetromino: function(stringOfCoordinates){
+		let math = stringOfCoordinates.split("\n");
+		let arrayOfCoordinates = [];
+		module.exports.getCoordinates(math);
```

Comment:
why are you invoking this function here but not doing anything with results?

Reply:
Why are you using function via module exports? Please invoke the function directly. And the next line as well.

Reply:
"why are you invoking this function here but not doing anything with results? "- I do not understand  this.
I work with function result.

"Why are you using function via module exports? Please invoke the function directly. And the next line as well." - I don't know how to do it

Reply:
> I work with function result.
Please specify exact line where you use result of function invoked on line 288

Reply:
> I don't know how to do it
OK